src/capi/azext_capi/custom.py

```diff
@@ -309,9 +369,34 @@ def create_workload_cluster(  # pylint: disable=unused-argument,too-many-argumen
         ephemeral_disks=False,
         windows=False,
         pivot=False,
-        output_path=None,
```

Comment:
Was dropping `output_path` intentional?

Reply:
yes, it is intentional `output_path` is not implemented yet and unused. Will be better to add it when needed